### ISSUE TITLE
FAPI_POS breaks Targets.pm parsing

### ISSUE THREAD

**Issue by spinler** [open-power/common-mrw-xml#5]
When I rebased with the latest XML and run a script that uses Targets.pm, I get:

Can't use string ("NA") as a HASH ref while "strict refs" in use at mrw/Targets.pm line 253.  That is: 
```
            $self->{enumeration}->{$enumType}->{$enum} =
              $baseptr->{$enumType}->{enumerator}->{$enum}->{value};
```
in function storeEnumerations.  I put in some traces and found it fails on the FAPI_POS attribute.  I'm not sure what's wrong with it myself...

**Comment by dcrowell77** [comment]
FAPI_POS will be required in the generated output that Hostboot sees, but that logic could be inside serverwiz2 itself versus being in the input system xml I suppose...

**Comment by spinler** [comment]
I don't care if it's there, it just can't crash the parser.

I imagine there is something wrong with this:

```
<enumerationType>
    <id>FAPI_POS</id>
    <description>Enumeration defining special FAPI_POS values</description>
    <enumerator>
        <name>NA</name>
        <value>0xFFFFFFFF</value>
    </enumerator>
    <default>NA</default>
</enumerationType>

<attribute>
    <id>FAPI_POS</id>
    <description>
...
```

**Comment by spinler** [comment]
I submitted a pull request that comments out the `enumerationType` element so I can get by it.

**Comment by nkskjames** [comment]
Fixed with serverwiz@d5439651a5c89c7a51038ce65c62ab8483271627
